src/errno.rs

```diff
@@ -48,6 +48,10 @@ cfg_if! {
         unsafe fn errno_location() -> *mut c_int {
             unsafe { libc::_Errno() }
         }
+    } else if #[cfg(target_os = "emscripten")] {
+        unsafe fn errno_location() -> *mut c_int {
+            unsafe { libc::__errno_location() }
+        }
```

Comment:
Looks like we can use Linux's `__errno_location()` as well

Reply:
Thank you for your review! I'll fix it!